staging/src/k8s.io/apiserver/pkg/cel/library/format.go

```diff
@@ -32,6 +31,13 @@ import (
 	"k8s.io/kube-openapi/pkg/validation/strfmt"
 )
 
+var (
+	// BASE64_LENGTH estimate for base64 regex size from github.com/asaskevich/govalidator
+	BASE64_LENGTH = 84
+	// URL_LENGTH estimate for url regex size from github.com/asaskevich/govalidator
+	URL_LENGTH    = 1103
```

Comment:
unless we *need* to export these, I'd probably just keep them private

Reply:
righto!